### PR TITLE
Script to run small version of prod pipeline with current branch

### PR DESCRIPTION
**Pull request by damonbayer**
Adds `test` argument to `main` function in `pipelines/batch/setup_prod_job.py`
This argument changes the default mcmc draws number, the locations processed, and storage for results.

This is then used in `pipelines/batch/setup_test_prod_job.py`, which runs the above function with `test=True` using the `pyrenew` container with tag of the current github branch, which is automatically created when that branch has an open PR.

### CODE REVIEW

File: pipelines/batch/setup_prod_job.py [hunk 1]
```diff
@@ -78,6 +79,12 @@ def main(
             f"supported diseases are: {', '.join(supported_diseases)}"
         )
 
+    pyrenew_hew_output_container = (
+        "pyrenew-test-output" if test else "pyrenew-hew-prod-output"
+    )
+    n_warmup = 10 if test else 1000
```
Comment (dylanhmorris) [inline pipelines/batch/setup_prod_job.py:85]: I think we may as well draw enough samples to make results are interpretable, even in test runs. Sampling time unlikely to be the main bottleneck in running this and it makes the test stronger.

File: pipelines/batch/setup_test_prod_job.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+from datetime import datetime, timezone
+from pathlib import Path
+
+from pygit2 import Repository
+from setup_prod_job import main
+
+if __name__ == "__main__":
+    current_datetime = datetime.now(timezone.utc).strftime("%Y%m%d%H%M%SZ")
+    current_branch = Path(Repository(os.getcwd()).head.name).stem
+    locs_to_exclude = [  # keep CA, MN, SD
```
Comment (dylanhmorris) [inline pipelines/batch/setup_test_prod_job.py:18]: And US, yes?

**Review by dylanhmorris — APPROVED** [review]
A few minor suggestions, none critical. Thanks, @damonbayer!

File: pipelines/batch/setup_test_prod_job.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+import os
+from datetime import datetime, timezone
+from pathlib import Path
+
+from pygit2 import Repository
+from setup_prod_job import main
+
+if __name__ == "__main__":
+    current_datetime = datetime.now(timezone.utc).strftime("%Y%m%d%H%M%SZ")
+    current_branch = Path(Repository(os.getcwd()).head.name).stem
```
[the remark this answers is not in the document]
Comment (damonbayer) [reply pipelines/batch/setup_test_prod_job.py:17]: I think I will make the branch name configurable instead.

**Pull request merged**